oauth2.md

```diff
@@ -165,7 +181,7 @@ Link a user id of your platform to Mocaverse.
 #### Request
 
 ```
-POST https://api.moca-id.mocaverse.xyz/api/oauth/partner/user
+POST https://oauth.mocaverse.xyz/api/oauth/partner/user
```

Comment:
think this remains as `api.moca-id`.

oauth is just to get the token

Reply:
For both request changes?

Reply:
yes, all API requests